### PR TITLE
doc(service): Add headers for getter and settter of model Id

### PR DESCRIPTION
**Pull request by bikamani**
Add headers for getter and settter of model Id

### CODE REVIEW

File: service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceTwinDevice.java [hunk 1]
```diff
@@ -644,10 +644,21 @@ private TwinCollection setToMap(Set<Pair> set)
         return map;
     }
 
+    /**
+     * Getter to get model Id
```
Comment (drwill-ms) [inline service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceTwinDevice.java:648]: Is this kind of language typical? "Getter to get" and "Setter to set" seems redundant.

File: service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceTwinDevice.java [hunk 2]
```diff
@@ -644,10 +644,21 @@ private TwinCollection setToMap(Set<Pair> set)
...
+     * @return The DTDL model Id of the device
+     * The value will be null for a non-pnp device.
+     * The value will be null for a pnp device until the device connects and registers with the model Id.
```
Comment (drwill-ms) [inline service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceTwinDevice.java:651]: with **a** model Id.

**Review by drwill-ms — APPROVED** [review]

File: service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceTwinDevice.java (hunk 1 above)
Comment (bikamani) [reply service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceTwinDevice.java:648]: Yes that's the language I have seen for other get and set method.

Comment (bikamani) [reply service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceTwinDevice.java]: addressed this.

File: service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceTwinDevice.java (hunk 2 above)
Comment (bikamani) [reply service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceTwinDevice.java]: done

File: service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceTwinDevice.java (hunk 1 above)
Comment (timtay-microsoft) [reply service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceTwinDevice.java]: Then the other places should be updated. This isn't a good description. "Gets the model Id" is much better

**Pull request merged**